service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -631,9 +631,6 @@ public ResponseEntity<ApiCloneWorkspaceResult> cloneWorkspace(
     String generatedDisplayName =
         sourceWorkspace.getDisplayName().orElse(sourceWorkspace.getUserFacingId()) + " (Copy)";
 
-    AzureCloudContext azureCloudContext =
```

Comment:
I am removing the optional AzureContext argument from cloning, as we intend to remove the `bpm_azure_enabled` entirely flag once the integration tests that rely on Azure managed app coordinates (VM tests) can be transitioned to using a spend profile from BPM.

I debated throwing an assertion in this method if `bpm_azure_enabled` is false, but given that this code path is also used for GCP workspace cloning (for which Azure-specific features and variables don't make any sense anyway), it seemed like somewhat of an odd thing to be checking. Hopefully `bpm_azure_enabled` will soon be gone.

Reply:
Does that mean you cannot express cloning across LZs through the REST API?

Reply:
You can-- when you clone the workspace, you provide the destination spend profile ID, and that's tied to a landing zone.